tasks/test_results_processor.py

```diff
@@ -360,14 +425,16 @@ def process_individual_arg(self, upload: Upload, repository) -> List[Testrun]:
         payload_bytes = archive_service.read_file(upload.storage_path)
         data = json.loads(payload_bytes)
 
-        testrun_list = []
+        parsing_results: list[ParsingInfo] = []
+
+        # TODO: this is bad
+        self.network = data.get("network_files")
```

Comment:
Can you add more context into why this is bad? I see that `network_files` is supposed to be in the uploaded test file, but I don't have much context into why getting this here is a bad thing?

Reply:
ah, this was a note for myself to revisit, the alternative to this is to return the network from the function, which i think makes more sense, but i wanted to quickly make sure it worked so i just set it as an attribute on the task itself, will fix this with the next commit